falco/values.yaml

```
@@ -12,7 +12,7 @@ image:
   # -- The image repository to pull from
   repository: falcosecurity/falco-no-driver
   # -- The image tag to pull. Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: "master"
```

```suggestion
  tag: ""
```

Please don't forget to revert this.